src/main/java/pl/gov/coi/pomocua/ads/dictionaries/domain/CityRepository.java

```diff
@@ -11,4 +12,7 @@
 
     @Query("SELECT c FROM City c WHERE UPPER(CONCAT(c.location.city, ', ', c.location.region)) LIKE UPPER(CONCAT(:query, '%'))")
     List<City> findByLocationCityWithPageable(@Param("query") String query, Pageable pageable);
```

Comment:
This method is unused now, right? If so, maybe let's delete it?

Reply:
done